app/components/object-inspector/sort-properties.js

```diff
@@ -6,10 +6,10 @@ import { A } from '@ember/array';
 export default Component.extend({
   tagName: '',
 
-  isArray: computed('properties', function () {
```

Comment:
Should we leave this computed and switch it to `'properties.[]` to ensure it updates?

Reply:
In this case I don't think it matters since it's only called once.